toolchain/testing/yaml_test_helpers.h

```diff
@@ -58,6 +58,19 @@
 
 namespace Carbon::Testing::Yaml {
 
+// Adds the specified indentation before each newline in the given string.
+inline auto IndentString(std::string_view str, std::string_view indent = "  ")
```

Comment:
Is the "indent" parameter needed?

(if so, you might consider llvm::StringRef for broader consistency, although I know some testing stuff uses string_view)